pkg/apiserver/handlers.go

```diff
@@ -402,8 +402,12 @@ func CORS(handler http.Handler, allowedOriginPatterns []*regexp.Regexp, allowedM
 				if allowedHeaders == nil {
 					allowedHeaders = []string{"Content-Type", "Content-Length", "Accept-Encoding", "X-CSRF-Token", "Authorization", "X-Requested-With", "If-Modified-Since"}
 				}
+				if exposedHeaders == nil {
+					exposedHeaders = []string{"Date"}
```

Comment:
Add a comment wherever we are adding the Date header saying if we remove that, we should remove this as well?